tests/ui/const-generics/issues/issue-68366.min.stderr

```diff
@@ -24,6 +29,11 @@ LL | impl<const N: usize> Foo {}
    |
    = note: expressions using a const parameter must map each value to a distinct output value
    = note: proving the result of expressions other than the parameter are unique is not supported
+help: Either remove the type parameter 'N' or make use of it, for example ` for S<N>`.
+   |
+LL - impl<const N: usize> Foo {}
+LL + impl<> Foo {}
```

Comment:
should remove the generic parameters entirely instead of leaving empty ones around